src/lib/InstantSearch.js

```diff
@@ -195,7 +211,9 @@ class InstantSearch extends EventEmitter {
         typeof widget.dispose !== 'function'
       ) {
         throw new Error(
-          'The widget you tried to remove does not implement the dispose method, therefore it is not possible to remove this widget'
+          `The \`dispose\` method is required to remove the widget.
+
+See: https://www.algolia.com/doc/guides/building-search-ui/widgets/create-your-own-widgets/js/`
```

Comment:
why not point directly the lifecycle sections by adding: `#the-widget-lifecycle-and-api` ?

Reply:
Indeed, I'll add the anchor.